src/components/SegmentedCard/SegmentedCard.jsx

```diff
@@ -21,11 +21,12 @@ class SegmentedCard extends React.Component<Props> {
     const className = cx(style.card, {
       [style.vertical]: this.props.vertical,
     });
+    const children = React.Children.toArray(this.props.children);
 
     return (
       <div {...getDataAttrs(this.props.data)} className={className} style={this.props.style}>
-        <div className={style.first}>{this.props.children[0]}</div>
-        <div className={style.second}>{this.props.children[1]}</div>
+        <div className={style.first}>{children[0]}</div>
+        <div className={style.second}>{children[1]}</div>
```

Comment:
You can't really ensure exactly two children, though...

Reply:
True. But I did not change from the old code